observations/sose_data.py

```diff
@@ -120,3 +121,53 @@ def compute_geostrophic_velocities(ds, lat, lon, day_offset, days, zF, α, β, g
 
     return U_geo, V_geo
 
+def plot_site_analysis(ds, lat, lon, day_offset, days):
```

Comment:
Is there a python ECCO analysis package? Feels like something that could go in there (not relevant to this PR, just musing...)

Reply:
Yes there's https://github.com/ECCO-GROUP/ECCOv4-py which I haven't used but they have extensive docs https://ecco-v4-python-tutorial.readthedocs.io/index.html# so could be a useful package when we move to using ECCO data.